lighthouse/hooks/events.py

```diff
@@ -0,0 +1,26 @@
+#  https://docs.python-eve.org/en/stable/features.html#database-event-hooks
+import logging
+from typing import Any, Dict, List
+
+from lighthouse.classes.automation_system import AutomationSystem
+from lighthouse.classes.biosero import Biosero
+
+logger = logging.getLogger(__name__)
+
+
+def inserted_events_hook(events: List[Dict[str, Any]]) -> None:
+    biosero = Biosero()
+    automation_system = AutomationSystem.AutomationSystemEnum.BIOSERO
+
+    for event in events:
+        event_type = event.get("event_type")
+
+        if event_type is None or not isinstance(event_type, str) or not event_type:
+            raise Exception("Cannot determine event type in hook")
+
+        logger.info(f"Attempting to publish a '{event_type}' plate event message from {automation_system.name}")
+
+        plate_event = biosero.get_plate_event(event_type)
```

Comment:
I would generate a new instance of biosero in each iteration, or call a method to reset() or clear() the contents, because at now all plate events are using the same "templates" and we risk leaving unwanted values for the next event.

Reply:
Yes, let's go for the simple solution and initialise on each event received. This should not happen in reality since we are only expecting one event at a time and have not published documentation of how to submit multiple events.